x-pack/plugins/fleet/server/services/agent_policy.test.ts

```diff
@@ -141,6 +147,148 @@ describe('agent policy', () => {
       const [, attributes] = soClient.create.mock.calls[0];
       expect(attributes).toHaveProperty('is_managed', true);
     });
+
+    it('should call audit logger', async () => {
```

Comment:
I started adding unit tests here for the core operations subject to audit logging. Will add this to all relevant test modules.